examples/android/src/main/AndroidManifest.xml

```diff
@@ -61,5 +60,7 @@
                 android:resource="@xml/provider_paths" />
         </provider>
     </application>
-
+    <queries>
+        <package android:name="com.dropbox.android" />
+    </queries>
```

Comment:
If we do https://github.com/dropbox/dropbox-sdk-java/pull/328, we could create a separate android AAR artifact that would include this in the library.  I think it may be about time for that. 🤷‍♂️